docs/Node Operators/performing-system-maintenance.md

```diff
@@ -87,23 +74,17 @@ cd ~/.chainlink-kovan && docker run -p 6688:6688 -v ~/.chainlink-kovan:/chainlin
 cd ~/.chainlink && docker run -p 6688:6688 -v ~/.chainlink:/chainlink -it --env-file=.env smartcontract/chainlink local n
 ```
 
-You should see the same `[INFO]` message that the node is waiting for lock on the db file. You may now shut down the second instance of the node and the original instance automatically obtains a lock and resumes normal operation.
+When the log messages on the first node indicate that it is waiting for the database lock, shut down the second instance of the node. The original instance automatically obtains a lock and resumes normal operation.
 
 ## Failover Node Example
 
 > 📘 Note
 >
 > This example uses Docker to run the Chainlink node, see the [Running a Chainlink Node](../running-a-chainlink-node/) page for instructions on how to set it up.
 
-You may want to run multiple instances of the Chainlink node on the same machine, so that if one instance goes down, the secondary instance can automatically pick up requests. Building off the concepts in the previous example, we'll use Docker to have primary and a secondary containers referencing the same database file.
-
-To begin, edit your environment variable file to set `DATABASE_TIMEOUT` to 0:
-
-```
-DATABASE_TIMEOUT=0
-```
+You might want to run multiple instances of the Chainlink node on the same machine. If one instance goes down, the second instance can automatically pick up requests. Building off the concepts in the previous example, use Docker to have primary and a secondary containers referencing the same database URL.
 
-This ensures that any secondary container will wait indefinitely for a lock on the database file.
+Always use `DATABASE_LOCKING_MODE=lease` unless you absolutely require compatibility with an older node version that is using advisory locks. See [the docs](/docs/configuration-variables/#database_locking_mode) for more information about this configuration variable.
```

Comment:
Should this be `DATABASE_LOCKING_MODE=advisorylock` for now?

Reply:
Updated the default and clarified best practices. See commit [32c596f](https://github.com/smartcontractkit/documentation/pull/558/commits/32c596f5365545fc2d3855a82d396c4cef6faf81) for the delta.